cogito/putter.go

```diff
@@ -51,13 +51,28 @@ func (putter *ProdPutter) LoadConfiguration(input []byte, args []string) error {
 		"environment", putter.Request.Env,
 		"args", args)
 
+	// If sink github is not requested, input dir is not mandatory.
+	// Validate the sinks first.
+	_, err = putter.Sinks()
+	if err != nil {
+		return fmt.Errorf("put: arguments: %w", err)
```

Comment:
this does not tell a lot. `put: load config: failed to find sinks?`

Reply:
The error is reported by `Sinks()` and as the name indicates, it's all about sinks.